test.js

```diff
@@ -37,6 +41,10 @@ test('matcher.isMatch()', t => {
 	t.true(matcher.isMatch('moo', ['*oo', '!f*']));
 	t.true(matcher.isMatch('UNICORN', ['!*oo', 'UNI*'], {caseSensitive: true}));
 
```

Comment:
Move these to a separate test with a descriptive title of what they're testing. Same with the other assertions.

Reply:
Did so. New tests named "... negated patern placement".